script/main.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func printUsageAndExit() {
-	log.Fatalf("Usage: ./processor <review> [--test-issue <issue name>]")
+	log.Fatalf("Usage: ./processor <review|approve> [--test-issue <issue name>]")
```

Comment:
[nit] Since there is nothing formatted, you can use `log.Fatal()` instead of `log.Fatalf()`